coresimd/arm/mod.rs

```diff
@@ -20,6 +15,7 @@ mod v7;
 #[cfg(any(target_arch = "aarch64", target_feature = "v7"))]
 pub use self::v7::*;
 
+// TODO move into the `acle::{dsp,simd32}` modules
```

Comment:
Can you perform this move as part of this PR?

Reply:
Will do later.